test/tests/button/happy.js

```diff
@@ -870,6 +870,58 @@ for (let flow of [ 'popup', 'iframe' ]) {
             }, '#testContainer');
         });
 
+        it('should render a button into a container and click on the button then call on shipping change', (done) => {
+
+            window.paypal.Button.render({
+
+                test: { flow, action: 'shippingChange' },
+
+                payment() : string | ZalgoPromise<string> {
+                    return generateECToken();
+                },
+
+                onAuthorize() : void {
+                    return done(new Error('Expected onAuthorize to not be called'));
+                },
+
+                onShippingChange() : void {
+                    return done();
+                },
+
+                onCancel() : void {
+                    return done(new Error('Expected onCancel to not be called'));
+                }
+
+            }, '#testContainer');
+        });
+
+        it('should render a button into a container and click on the button then call reject on shipping change', (done) => {
+
+            window.paypal.Button.render({
+
+                test: { flow, action: 'shippingChange' },
+
+                payment() : string | ZalgoPromise<string> {
+                    return generateECToken();
+                },
+
+                onAuthorize() : void {
+                    return done(new Error('Expected onAuthorize to not be called'));
+                },
+
+                onShippingChange(data, actions) : void {
+                    return actions.reject(() => {
```

Comment:
I think we can just do `actions.reject().then(() => done())` here right? Then it's a little closer to the actual usage of reject.